docs/release-notes/8.18.asciidoc

```diff
@@ -1,6 +1,29 @@
 [[release-notes-header-8.18.0]]
 == 8.18
 
+[discrete]
+[[release-notes-8.18.8]]
+=== 8.18.8
+
+[discrete]
+[[features-8.18.8]]
+==== New features
+* Adds an {elastic-defend} option to remediate orphaned state by attempting to start Elastic Agent service.
+
+[discrete]
+[[enhancements-8.18.8]]
+==== Enhancements
+* Fixes {elastic-defend} error log on Windows where only the first character, usually 'C', was logged instead of a path.
+
+[discrete]
+[[bug-fixes-8.18.8]]
+==== Fixes
+* Removes `null` in confirmation dialog when bulk editing index patterns for rules ({kibana-pull}236572[#236572]).
+* Fixes the URL passed to detection rule actions via the `{{context.results_link}}` placeholder ({kibana-pull}236067[#236067]).
+* Fixes a bug in {elastic-defend} where Linux network events could have source and destination bytes swapped.
```

Comment:
```suggestion
* Adds support in {elastic-defend} for installing eBPF probes on Linux endpoints when taskstats is compiled out of the kernel.
* Fixes a bug in {elastic-defend} where Linux network events could have source and destination bytes swapped.
```
Add back in the removed note because I just discovered it's actually in the BC build from last night.
https://buildkite.com/elastic/unified-release-staging/builds/3354#0199a3ba-27e6-462c-b4a9-220a8d4bdfe1/101-123